packaging/rpm/Makefile

```diff
@@ -30,8 +33,10 @@ srpm: build_prepare
 		-r $(MOCK_CONFIG) \
```

Comment:
```suggestion
		-r $(MOCK_CONFIG) \
		--bootstrap-chroot
```

Reply:
I read up a little bit more about the "RHEL7 can't build EL8 RPMs" here: https://github.com/rpm-software-management/mock/wiki/FAQ#i-cannot-build-fedora-or-rhel8-beta-package-on-rhelcentos-7

And this was the solution I found to work.